buildspec.yml

```diff
@@ -1,10 +1,30 @@
 # AWS CodeBuild config
 
-# For now, this is just a placeholder to test the CodeBuild setup
+# Builds the backend site with Docker and the image to ECR. 
+# TODO: add public and private sites.
 
 version: 0.2
 
+env:
+  variables:
+    IMAGE_TAG: "latest-test" # TODO: change to latest?
```

Comment:
We'll actually have at least a couple of images per service.  One will be set up for testing and another will be optimized for production.  They will use the same Dockerfile but have different targets.  I've been using "test" and "run" tags for those, respectively, but that's not a hard requirement.  We'll actually probably want to use hash-based tags until we're ready to promote as part of the final deployment step, but again, that can easily be added later.